lib/match.js

```diff
@@ -25,10 +25,19 @@ aliases['kebab-case'] = aliases.kebabcase;
 
 const getRegex = (value, filename) => {
   if (value instanceof RegExp) return [value, value.toString()];
+
   if (typeof value === 'string') {
-    const regex = aliases[value];
-    if (!regex) throw new Error(`Unrecognized option "${value}"`);
-    return [regex, value];
+    const predefinedRegex = aliases[value];
+    if (predefinedRegex) return [predefinedRegex, value];
+
+    try {
+      const regex = RegExp(value);
+      if (regex instanceof RegExp) return [regex, regex.toString()];
+    } catch (e) {
+      // Ignore to fallback into the same error message as the RegExp constructor
+    }
```

Comment:
Sorry about the long radio silence. How about this?
```suggestion
  if (value[0] === '/') {
    const ix = value.lastIndexOf('/');
    if (ix > 0) {
      const regex = new RegExp(value.substr(1, ix - 1), value.substr(ix + 1));
      return [regex, value];
    }
  }
```